specs/agents/tracing-api-otel.md

```diff
@@ -32,7 +32,7 @@ The OTel API/specification goes beyond tracing, as a result, the following OTel
 - metrics
 - logs
 - span events
-- span links
+- span link *attributes*
```

Comment:
@SylvainJuge I'm not sure there is anything more to change in the OTel Bridge spec. I'm propose here that implementations of the OTel Bridge change to support span links -- although any given span link attributes would be dropped.  Do you think more is required in this markdown file for that?

Reply:
I think it's enough to change only this for now.